src/_angular/file-selector/dtv-file-selector.js

```diff
@@ -204,6 +198,12 @@
                 scope.selectorValid = false;
               }
 
+              if (!scope.subscriptionOff && !scope.hideSubscription) {
+                // ensure subscription-status component does a refresh in case user subscribed from in-app storage
+                scope.isSubscribed = false;
```

Comment:
@rodrigopavezi Curious, won't this briefly show the component on the page even if the user is subscribed since the component has to do an api call and get a response?